babeld/babel_interface.c

```diff
@@ -606,6 +570,25 @@ DEFUN (babel_set_channel_noninterfering,
     return CMD_SUCCESS;
 }
 
+DEFPY (no_babel_set_channel,
+       no_babel_set_channel_cmd,
```

Comment:
Can't this be combined with `babel_set_channel_cmd` too?

Reply:
I planned to do a minimal change, however it is already large.
Now I merged existing `babel channel *` commands into one single function.